Scripts/RemediationScripts/Remediate-SetDBForPostgreSQLSingleServerMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,847 @@
+<###
+# Overview:
+    This script is used to set minimium required TLS version and enable ssl enforcement for Azure Database for PostgreSQL single server in a Subscription.
+# Control ID:
+    Azure_DBforPostgreSQL_DP_Use_Secure_TLS_Version
```

Comment:
NIT: Check for such new lines and maintain consistency across the file

Reply:
fixed.